pipeline/measuring.py

```diff
@@ -93,6 +92,26 @@ def __init__(self, **kwargs):
             'to compare against the real width (x1.0) when running psf width filter. '
         )
 
+        self.thresholds = self.add_par(
+            'thresholds',
+            {
+                'negatives': 0.3,
+                'bad pixels': 1,
+                'offsets': 5.0,
+                'filter bank': 1,
+            },
+            dict,
+            'Thresholds for the disqualifier scores. '
+            'If the score is higher than the threshold, the measurement is disqualified. '
```

Comment:
Good catch. It does indeed have to be larger or equal, because of integer or boolean thresholds. 
For example, if your score is true/false then your threshold should be true, or in numeric terms 1.